src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -143,7 +142,7 @@ internal PackageSourceMoniker SelectedSource
 
             SetTitle();
 
-            _topPanel.IsSolution = Model.IsSolution;
+            _topPanel.CreateAndAddConsolidateTab(Model.IsSolution);
```

Comment:
this method probably doesn't need to be called if IsSolution is false. (make sure topPanel has right default for IsSolution in case it doesn't get called.